settings/global.mdx

```diff
@@ -66,15 +66,29 @@ settings. Learn more about the [properties](#properties) or from an
 </ResponseField>
 
 <ResponseField name="theme" type={'"venus" | "quill" | "prism"'}>
-  A preset theme configuration that changes the layout of your docs. Examples:
+  A preset theme configuration that changes the look and feel of the project. A theme is a set of default styling configurations.
+  
+  Examples:
   [Venus](https://starter-venus.mintlify.app),
   [Quill](https://starter-quill.mintlify.app),
   [Prism](https://starter-prism.mintlify.app)
 </ResponseField>
 
+<ResponseField name="layout" type={'"topnav" | "sidenav" | "solidSidenav"'} default="topnav">
+  The layout of the project. Setting it to `topnav` will feature the top bar as the primary navigation. Setting it to `sidenav` will feature the sidebar. The `solidSidenav` will have an active sidenav.
+</ResponseField>
+
+<ResponseField name="background" type="Background">
+  Set a decorative background
+  <Expandable title="Background">
+    <ResponseField name="styling" type={'"gradient" | "grid" | "windows"'}>
+      The style of the decorative background
```

Comment:
i think we should maybe have links to examples for this.

Reply:
Don't have a ton of examples atm. We can build examples and then link back here